mount/mount_windows.go

```diff
@@ -84,9 +84,8 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 		allOptions = append(allOptions, options...)
 		allOptions = append(allOptions, sensitiveOptions...)
 		if len(allOptions) < 2 {
-			klog.Warningf("mount options(%q) command number(%d) less than 2, source:%q, target:%q, skip mounting",
+			return fmt.Errorf("mount options(%q) should have at least 2 options, current number:%d, source:%q, target:%q",
 				sanitizedOptionsForLogging, len(allOptions), source, target)
```

Comment:
It is up to the called to make sure using senstiveOptions for password, but it there a way we can verify that?

Reply:
actually `sensitiveOptions` is not used on Windows:
```
func (mounter *Mounter) Mount(source string, target string, fstype string, options []string) error {
	return mounter.MountSensitive(source, target, fstype, options, nil /* sensitiveOptions */)
}
```

Reply:
Is that a concern?

Reply:
@msau42 I don't think so, windows has different implementation with linux, and as long as we don't print password in logging, that would be ok

Reply:
actually could you verify that whether it will be printed out or not?
The caller (driver) should call MountSensitive instead of Mount if it has password information, I think.

Reply:
I checked the code again, use `MountSensitive`  func won't print password.

Reply:
thanks. After kubernetes/kubernetes#94126 is merged, we can merge this.

Reply:
hi @jingxu97 that's not related, `len(allOptions)` would be changed by https://github.com/kubernetes/kubernetes/pull/94126

Reply:
I see. thanks!